packages/fast-components-react-base/src/checkbox/checkbox.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Checkbox",
+    "description": "A checkbox component's schema definition.",
+    "type": "object",
+    "id": "checkbox",
+    "properties": {
+        "tag": {
+            "title": "HTML element tag",
```

Comment:
nit: prefer `HTML tag` (how it is on current fluentweb)

Reply:
fixed